### ISSUE TITLE
Reg.multi-task fine-tuning

### ISSUE THREAD

**Issue by mirishkarganesh** [Open-Speech-EkStep/vakyansh-wav2vec2-experimentation#20]
Hi Folks,

I am looking for multi-task fine-tuning. 
Can you please help with where can I make necessary changes for including "training loss" (which would be the weighted sum of the losses of the tasks.)

**Comment by harveenchadha** [comment]
Can you please elaborate what do you mean by multi-task fine tuning?

**Comment by mirishkarganesh** [comment]
Hi @harveenchadha ,

I am looking for Joint training something like LID +ASR.
So for this kind of task we need to have joint loss right.
So I am looking for such kind of loss function in w2v.
Can you please help me with it?

**Comment by sridhar1ga** [comment]
Hello @harveenchadha
I'm also trying to do the same, Can you please help me out.

Thanks in Advance
